### PR TITLE
Views

### PR DESCRIPTION
**Pull request by erickma1**
In this milestone, I created the following;

- Basic UI created for pages to show all users fetched from database
- Basic UI created for pages to show all Posts fetched from database

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
...
+            </div>
+        </div>
+    <% end %>
+
+    <div class="row mt-4 justify-content-center">
+        <div class="col-lg-12 text-center">
+            <%= link_to "View All Posts", user_posts_path(@user), class: "btn btn-primary" %>
+        </div>
+    </div>
+</section>
```
Comment (mckent05) [inline app/views/users/show.html.erb:25]: - You have done well so far, according to the project requirements, your design should match the given wireframe. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. 

Wireframe:
wwegfae

Your project:
 - Please add the user info section for this page ( user profile picture and username)
4gw4ttq4

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,20 @@
...
+        <div class="row">
+            <div class="col-lg-12 border border-dark">
+                <% post.recent_comments.each do |comment| %>
+                    <p class=""><%= comment.author.name %>: <%= comment.body %> </p>
+                <% end %>
+
+            </div>
+        </div>
+    <% end %>
+</section>
```
Comment (mckent05) [inline app/views/posts/index.html.erb]: - You have done well so far, according to the project requirements, your design should match the given wireframe. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Wireframe:
rgwqrtgqw4

Your project:
 - Please add the user info section for this page (username and user photo) 
rgawrgqaw

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @erickma1, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have done a good job creating view files for your application 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
Changed design should match the given wireframe

**Review by mikethreels — APPROVED** [review]
Hi Eric, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the requested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**